pandas/core/generic.py

```diff
@@ -4232,7 +4232,35 @@ def as_matrix(self, columns=None):
 
     @property
     def values(self):
-        """Numpy representation of NDFrame
+        """
+        Generate and return a Numpy representation of NDFrame.
+
+        Only the values in the NDFrame will be returned, the axes labels will
```

Comment:
> The original docstring contained the private class "NDFrame", so I kept it in there despite failing the docstring test.

Yes, I know there are occurrences of NDFrame in the current docstrings, but that is exactly why we added the check in the validation script :). As that name should never be exposed to users. 

Now in this specific case: this docstring is actually only used for DataFrame (Series has a separate docstring), so you can just use "DataFrame" instead of "NDFrame" 
(it is also used for Panel, but since that is deprecated, it is not really a problem that its docstring is not fully correct)

Reply:
I had noticed the inheritances - thank you for the clarification; I made the recommended changes